charts/rancher-vsphere-csi/rancher-vsphere-csi/2.3.0/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```

Comment:
This one I actually think might need to stay ? @PennyScissors when we OOB these they will be 100.x.x right ? If so then this is good.